src/main/java/com/marksem/crm/entity/TypeContact.java

```diff
@@ -0,0 +1,7 @@
+package com.marksem.crm.entity;
+
+public enum TypeContact {
+    MobilePhone,
```

Comment:
Call all enum values as constants - uppercase with underscores to separate words